docs/2.8.0/docs/daml-finance/index.rst

```diff
@@ -37,6 +37,7 @@ The following is a suggested learning path to get productive quickly:
 #. Learn how to
    :doc:`use the instrument packages to model different financial instruments <instruments/intro>`
 #. Explore the :ref:`Daml Finance Demo Application <daml-finance-demo-app>`
+#. Check out the `Forum <https://discuss.daml.com/tag/daml-finance>`_
```

Comment:
I woiuld explain what the forum is meant for. Maybe outside of the bullets something like "You can use the Daml forum to ask questions directly to the team" or something like that